### PR TITLE
Bug fix/several fixes

### PR DESCRIPTION
**Pull request by rem1niscence**

### CODE REVIEW

File: frontend/src/views/Apps/AppPassphrase/AppPassphrase.js [hunk 1]
```diff
@@ -96,12 +96,15 @@ class AppPassphrase extends Component {
...
+      const appData = PocketApplicationService.getApplicationInfo().data;
+
       PocketApplicationService.removeAppInfoFromCache();
       PocketApplicationService.saveAppInfoInCache({
         applicationID: applicationInfo.id,
         passphrase: passPhrase,
         address,
-        privateKey
+        privateKey,
+        data: appData
```
Comment (xerp) [inline frontend/src/views/Apps/AppPassphrase/AppPassphrase.js]: Here, you are duplicating the saved data, at this moment the only data that we don't save is the chains, and this field(chains) is not required at this moment.

File: frontend/src/views/Apps/AppDetail/AppDetail.js [hunk 2]
```diff
@@ -62,7 +63,8 @@ class AppDetail extends Component {
       networkData,
     } = await ApplicationService.getApplication(address);
 
-    const chains = await NetworkService.getNetworkChains(networkData.chains);
+    const allChains = await NetworkService.getAvailableNetworkChains();
+    const chains = allChains.map(addIndex);
```
Comment (xerp) [inline frontend/src/views/Apps/AppDetail/AppDetail.js]: Please, change the service method to encapsulate this behavior, add a parameter called **indexed**

File: frontend/src/core/services/PocketAccountService.js [hunk 3]
```diff
@@ -7,21 +7,27 @@ export class PocketAccountService extends PocketBaseService {
   }
 
   /**
-   * Import an account into network.
+   * Validatw an account into network.
```
Comment (xerp) [inline frontend/src/core/services/PocketAccountService.js]: Fix this typo.

File: frontend/src/core/components/DefaultLayout/AppSidebar/AppSidebar.scss [hunk 4]
```diff
@@ -47,10 +47,14 @@
...
 .app-sidebar-footer {
-  position: relative;
-  bottom: -290px;
+  position: absolute;
+  bottom: 10px;
+  margin: 0 auto;
+  width: 100%;
+
+  div.footer {
+    // margin-top: ;
```
Comment (xerp) [inline frontend/src/core/components/DefaultLayout/AppSidebar/AppSidebar.scss]: Remove this.

**Review by xerp — CHANGES_REQUESTED** [review]

File: frontend/src/views/Apps/AppDetail/AppDetail.js [hunk 5]
```diff
@@ -62,7 +62,7 @@ class AppDetail extends Component {
       networkData,
     } = await ApplicationService.getApplication(address);
 
-    const chains = await NetworkService.getNetworkChains(networkData.chains);
+    const chains = await NetworkService.getAvailableNetworkChains();
```
Comment (xerp) [inline frontend/src/views/Apps/AppDetail/AppDetail.js]: Possible issue here, here is not available networks, here is the networks of the app.

**Review by xerp — CHANGES_REQUESTED** [review]

**Review by xerp — APPROVED** [review]

File: frontend/src/views/Apps/AppPassphrase/AppPassphrase.js (hunk 1 above)
Comment (rem1niscence) [reply frontend/src/views/Apps/AppPassphrase/AppPassphrase.js]: Yeah, I removed the duplication, such information was added on the previous view

**Pull request merged**